### PR TITLE
Controllers and views added

### PR DESCRIPTION
**Pull request by hpavlo**

### CODE REVIEW

File: src/main/webapp/web.xml [hunk 1]
```diff
@@ -11,4 +11,49 @@
...
+        <url-pattern>/drivers</url-pattern>
+    </servlet-mapping>
+
+    <servlet>
+        <servlet-name>createDriver</servlet-name>
+        <servlet-class>mate.controller.CreateDriverController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>createDriver</servlet-name>
+        <url-pattern>/drivers/create</url-pattern>
```
Comment (CyberDogFK) [inline src/main/webapp/web.xml:30]: image
Change url-s according to common mistakes

**Review by CyberDogFK — CHANGES_REQUESTED** [review]

**Review by CyberDogFK — APPROVED** [review]
Looks good

File: src/main/java/mate/controller/AddDriverToCarController.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+    private final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/cars/drivers/addDriverToCar.jsp")
```
Comment (ChumakAn) [inline src/main/java/mate/controller/AddDriverToCarController.java:23]: could be a constant

**Review by ChumakAn — APPROVED** [review]
Looks goot to me

**Pull request closed without merging**